cbor/src/main/java/com/fasterxml/jackson/dataformat/cbor/CBORGenerator.java

```diff
@@ -115,6 +115,23 @@ public enum Feature implements FormatFeature {
          * @since 2.15
          */
         WRITE_MINIMAL_DOUBLES(false),
+
+        /**
+         * Feature that determines how binary tagged negative BigInteger values are
+         * encoded.
+         *
+         * When enabled, Ensures proper encoding of negative values
+         * (e.g., -1 is encoded [0xC3, 0x41, 0x00])
+         *
+         * When disabled, Maintains backwards compatibility with existing implementations
+         * (e.g., -1 is encoded [0xC3, 0x41, 0x01])
+         *
+         * The default value is false for backwards compatibility.
+         *
+         * @since 2.20.0
+         */
+
+        CORRECT_CBOR_NEGATIVE_BIGINT_ENCODING(false)
```

Comment:
I think I'll rename this to something like ENCODE_USING_STANDARD_BIGINT_ENCODING.